### PR TITLE
Dialogue

### PR DESCRIPTION
**Pull request by bcvery1**
There is still a z-index issue, text not appearing in the right order, but this can be reviewed as is stable

### CODE REVIEW

File: dialogue/dialogue.go [hunk 1]
```diff
@@ -0,0 +1,145 @@
...
+	currentDialogue []Dialogue
+	// dialoguePanel is the index of dialogue currently playing
+	dialoguePanel int
+
+	dialoguePanelSize = pixel.V(dialogueWidth, dialogueHeight)
+
+	textColour                            = color.RGBA{0x00, 0x00, 0xf1, 0x00}
+	dialoguePanelBackgroundColour         = color.RGBA{0xf3, 0xff, 0xf1, 0x00}
+	dialoguePanelBorderColour             = color.RGBA{0x00, 0x00, 0xf1, 0x00}
+	borderThinkness               float64 = 2
```
Comment (PhantomRaviolis) [inline dialogue/dialogue.go]: Did you mean thickness?

Comment (bcvery1) [reply dialogue/dialogue.go]: Yes - fixed in 0b9d0f7

File: dialogue/dialogue.go [hunk 2]
```diff
@@ -0,0 +1,145 @@
...
+
+	imd := imdraw.New(nil)
+	// Background
+	imd.Color = dialoguePanelBackgroundColour
+	imd.Push(posMin, posMin.Add(dialoguePanelSize))
+	imd.Rectangle(0)
+	// Border
+	imd.Color = dialoguePanelBorderColour
+	imd.Push(
+		posMin.Sub(pixel.V(borderThinkness, borderThinkness)),
```
Comment (PhantomRaviolis) [inline dialogue/dialogue.go]: I'm sure you will - but make sure to change all instances of var name, unless you want to Ryan it.

Comment (bcvery1) [reply dialogue/dialogue.go]: Yeah, have changed all instances and run the game with success

**Pull request merged**